packages/core/src/app/ui/accordion/AccordionItem.tsx

```diff
@@ -46,7 +46,6 @@ const AccordionItem: FunctionComponent<AccordionItemProps> = ({
             className,
             { [classNameSelected]: isSelected }
         ) }
-        data-test={ `accordion-item_${itemId}` }
```

Comment:
This is not working. `playwright` will not select a payment method after clicking this li.
It was working in Apple Pay e2e test because Apple Pay is the only option in the payment list.

Apple Pay test recording:

[e004646597fb864c31553020e45eae64.webm](https://user-images.githubusercontent.com/88361607/188043436-0cdc071b-21a1-4e0b-8536-be8375b04d82.webm)